Added Password Security and FileNotFound Exception Handling

H00pyFr00d opened the pull request:
Using the Keyring package to store passwords securely rather than storing them in a plaintext file

If a `FileNotFound` exception would be raised when looking for the username, the file is created and the user is prompted to give their username

H00pyFr00d committed:
Keyring Support + FileNotFound handling

Changed from storing passwords in a plaintext file to storing them using the python keyring

Also will create the userinfo.txt file if it doesn't exist (storing the username only now)

H00pyFr00d committed:
Update README.md

H00pyFr00d committed:
Reordered Code

Old code order could cause minor annoyances and potential errors with connections interrupting username and password entry

H00pyFr00d committed:
Merge branch 'password-security' of https://github.com/H00pyFr00d/edinburgh-uni-learn-login into password-security

bvechiato commented on main.py:
```diff
@@ -1,11 +1,29 @@
+import os
 
+import keyring
 from selenium import webdriver
```
`from seleniumwire import webdriver` removes the deprecated message

bvechiato committed:
Merge branch 'main' into password-security

H00pyFr00d committed:
Added Error Handling for No Internet

Added error handling so the program doesn't crash when launched with no internet connection

H00pyFr00d committed:
Merge branch 'password-security' of https://github.com/H00pyFr00d/edinburgh-uni-learn-login into password-security

H00pyFr00d committed:
Added GUI Elements

Implemented basic GUI elements using tkinter for getting username and password from the user

bvechiato reviewed (commented):
Works really well (although I still get a deprecated error message???), should we keep password/UI/main in separate files? It'd make it easier to maintain
